### PR TITLE
UI 

### PR DESCRIPTION
**Pull request by saketkum11**
- UI design for  Pomodoro app

### CODE REVIEW

**Review by Sharmaryan — APPROVED** [review]
You can merge

File: src/Routes/MyRoutes.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import Mockman from "mockman-js";
+const MyRoutes = () => {
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<Timer />} />
+        <Route path="/dashboard" element={<Dashboard />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<Signup />} />
+        <Route path="/mock" element={<Mockman />} />
```
Comment (Sharmaryan) [inline src/Routes/MyRoutes.js:12]: Why are you using mockman, i am asking this because we don't have APIs for pomodoro.

**Review by Sharmaryan — APPROVED** [review]
Clean code and UI

File: src/Routes/MyRoutes.js (hunk 1 above)
Comment (saketkum11) [reply src/Routes/MyRoutes.js:12]: Yes I got it.
I don't have to use this.

**Review by Pratik1005 — APPROVED** [review]
Nice UI, good to merge

**Review by shubhamkumar648 — APPROVED** [review]
- simple and clean ui
- good to merge

**Pull request merged**